packages/fxa-settings/src/components/AppLayout/index.tsx

```diff
@@ -132,14 +132,13 @@ export const AppLayout = ({
           </section>
         </main>
       </div>
-      <footer>
-            {/* LocaleToggle positioned in lower left corner of page */}
-            {showLocaleToggle && (
-              <div className="fixed bottom-6 left-6 z-10">
-                <LocaleToggle placement="footer" />
-              </div>
-            )}
+      {showLocaleToggle && (
+        <footer>
+          <div className="fixed bottom-6 left-6 z-10">
+            <LocaleToggle placement="footer" />
```

Comment:
Should we go ahead and remove `placement="footer"` too since it's not used in the component, and delete `placement` from the component props? As well as `className`?